Felienne/hedy
# [BUG FIX] Program submission works again

**Description**
In this PR we fix program submission which was program due to still being dependent on `auth.profile`, a front-end variable that keeps track on the user being logged in or not. We removed this but forgot it at this spot.

**Fixes**
This PR fixes #2212

**How to test**
Make sure you have a shared program. Try to submit it and verify this is working as expected.
